dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -88,6 +89,15 @@ internal class RumActionScope(
 
     // region Internal
 
+    private fun onStartView(
+        now: Long,
+        writer: DataWriter<Any>
+    ) {
+        // another view starts, complete this action
+        ongoingResourceKeys.clear()
```

Comment:
this means that an ongoing resource which was start by this action will be associated with the new action right ? Are we ok with this ?

Reply:
Yes, it will be. Not sure if it is the best logic, but it is basically exactly the same as in the `RumActionScope#onStopView`.